plugins/logitech-bulkcontroller/fu-logitech-bulkcontroller-device.c

```diff
@@ -1252,14 +1260,6 @@ fu_logitech_bulkcontroller_device_setup(FuDevice *device, GError **error)
 		return FALSE;
 	}
 
-	/* the hardware is unable to handle requests -- firmware issue */
-	if (fu_device_has_private_flag(device,
-				       FU_LOGITECH_BULKCONTROLLER_DEVICE_FLAG_POST_INSTALL)) {
-		fu_device_sleep(device, POST_INSTALL_SLEEP_DURATION);
-		fu_device_remove_private_flag(device,
-					      FU_LOGITECH_BULKCONTROLLER_DEVICE_FLAG_POST_INSTALL);
-	}
```

Comment:
IIRC I did test moving this sleep with poor results, at least on the Rallybar/Mini devices. In my testing there was a definite need for some kind of sleep in this location. It might just be an issue with the debug devices, though, as they tend to be slower.